### DISCUSSION TITLE
Support for `${userHome}` in settings

### DISCUSSION THREAD

**Discussion by paulober** [microsoft/pylance-release#5988] [Enhancement]
Please add support for `${userHome}` in `python.analysis.typeshedPaths` and `python.analysis.extraPaths` like VSCode has for its settings and with update 1.20 cpp-tools will also support it in c_cpp_properties.

**Comment by paulober** [comment]
@KacieKK any updates on this?

**Comment by KacieKK** [comment]
Hey @paulober, thanks for suggeting this. We have added this to our enhancement list. In the meantime, you could work this around by using `env:Home` .

**Reply by paulober** [reply]
Right, the problem is, `env:Home` isn't cross platform as Windows uses `env:USERPROFILE`.

**Reply by paulober** [reply]
and `${env:USERPROFILE}` doesn't work in `python.analysis.typeshedPaths`
